packages/extensions/openapi-to-typespec/src/resource/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const ProvidersSegment = "/providers/";
+export const ManagementGroupPath = "/providers/Microsoft.Management/managementGroups/{managementGroupId}";
+export const ManagementGroupScopePrefix = "/providers/Microsoft.Management/managementGroups";
+export const ResourceGroupPath = "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}";
+export const ResourceGroupScopePrefix = "/subscriptions/{subscriptionId}/resourceGroups";
+export const SubscriptionPath = "/subscriptions/{subscriptionId}";
+export const SubscriptionScopePrefix = "/subscriptions";
+export const TenantPath = "";
+export const TenantScopePrefix = "/tenants";
```

Comment:
i kind of doubt if we have such cases with this prefix.
For these hard-coded paths, could we have a link of where this comes from? such as a link to the code in our csharp generator.

Reply:
Just borrowed from .net generator. But I think we could refine it.